docs/man_pages/lib-management/plugin-add.md

```diff
@@ -1,50 +1,68 @@
-Add a plugin and set all variables from the command line | `$ appbuilder plugin add <Name or ID> --var.debug.<Variable ID> <Variable Value> --var.release.<Variable ID> <Variable Value>`[\*\*](#note)
-Add a specific version of a plugin and set all variables from the command line | `$ appbuilder plugin add <Name or ID>@<Version> --var.debug.<Variable ID> <Variable Value> --var.release.<Variable ID> <Variable Value>`[\*\*](#note)
-Add the latest version of a plugin and set all variables from the command line | `$ appbuilder plugin add <Name or ID> --latest --var.debug.<Variable ID> <Variable Value> --var.release.<Variable ID> <Variable Value>`[\*\*](#note)
-Add the default version of a plugin and set all variables from the command line | `$ appbuilder plugin add <Name or ID> --default --var.debug.<Variable ID> <Variable Value> --var.release.<Variable ID> <Variable Value>`[\*\*](#note)
+List Apache Cordova plugins | `$ appbuilder plugin add --available [--debug] [--release]`
+Add Apache Cordova plugins | `$ appbuilder plugin add <Name or ID> [--debug] [--release]`
+Add a specific version of an Apache Cordova plugin | `$ appbuilder plugin add <Name or ID>@<Version> [--debug] [--release] [--var.<Variable ID> <Variable Value>]`[\*\*](#note)
+Add the latest version of an Apache Cordova plugin | `$ appbuilder plugin add <Name or ID> --latest [--debug] [--release] [--var.<Variable ID> <Variable Value>]`[\*\*](#note)
+Add the default version of an Apache Cordova plugin | `$ appbuilder plugin add <Name or ID> --default [--debug] [--release] [--var.<Variable ID> <Variable Value>]`[\*\*](#note)
+Add an Apache Cordova plugin and set all variables from the command line | `$ appbuilder plugin add <Name or ID> --var.debug.<Variable ID> <Variable Value> --var.release.<Variable ID> <Variable Value>`[\*\*](#note)
+Add a specific version of an Apache Cordova plugin and set all variables from the command line | `$ appbuilder plugin add <Name or ID>@<Version> --var.debug.<Variable ID> <Variable Value> --var.release.<Variable ID> <Variable Value>`[\*\*](#note)
+Add the latest version of an Apache Cordova plugin and set all variables from the command line | `$ appbuilder plugin add <Name or ID> --latest --var.debug.<Variable ID> <Variable Value> --var.release.<Variable ID> <Variable Value>`[\*\*](#note)
+Add the default version of an Apache Cordova plugin and set all variables from the command line | `$ appbuilder plugin add <Name or ID> --default --var.debug.<Variable ID> <Variable Value> --var.release.<Variable ID> <Variable Value>`[\*\*](#note)
 
 <% if(isHtml) { %><a id="note"></a><% } %>
-\*\* If the plugin has multiple variables, you can set `--var` for each variable.
+\*\* If the Apache Cordova plugin has multiple variables, you can set `--var` for each variable.
+<% } %>
+
+<% if((isConsole && isMobileWebsite) || isHtml) { %>## NativeScript Projects<% } %>
```

Comment:
The check is isMobileWebsite but the text is >## NativeScript Projects - which one is correct?